b3sum/tests/test.rs

```diff
@@ -11,7 +11,19 @@ pub fn b3sum_exe() -> PathBuf {
 fn test_hash_one() {
     let expected = blake3::hash(b"foo").to_hex();
     let output = cmd!(b3sum_exe()).stdin_bytes("foo").read().unwrap();
-    assert_eq!(&*expected, &*output);
+    assert_eq!(&*expected, output);
+}
+
+#[test]
+fn test_hash_one_raw() {
+    let expected = blake3::hash(b"foo").as_bytes().to_owned();
+    let mut stdout = Vec::new();
+    let mut output_reader = cmd!(b3sum_exe(), "--raw")
+        .stdin_bytes("foo")
```

Comment:
You could also use `stdout_capture` here, if you wanted to save yourself the `read_to_end` call. Either way.

Reply:
Done.  Nice!